.travis.yml

```diff
@@ -31,3 +32,6 @@ script:
   - go vet ./management/...
   - test -z "$(golint ./Gododir/... | tee /dev/stderr)"
   - go vet ./Gododir/...
+  - test -z "$(set -e; for v in ./arm... ./datalake-store/... ./Gododir/...; do (errcheck -ignoretests "$v" | grep -v 'resp.body.Close()' | tee /dev/stderr); done)""
```

Comment:
1) if you will have test -z, set -e is not necessary.

2) instead of this grep hack, perhaps you can do:

```go
defer func(){
     _ = resp.Body.Close()
}()
```

3) can you please check all callers of `getRequestBody()` make sure that error from the request is nil and then go on reading the body? (if error from http.Do/Get/... is non-nil, then resp.Body will be nil, causing the panic in `defer resp.Body.Close()`

4) You have `grep -v 'resp.body.Close()'`  but I think `body` should be `Body`?